### PR TITLE
Fixes login UI for small popup window sizes

### LINKED ISSUE #862: Login page is too long for login popup
The UI changes in #842 have made the login page too long for the popup window:

screen shot 2018-10-13 at 21 30 35

Rather than making the popup window larger (it feels about the right size), we should have responsive CSS that saves vertical space on small vertical window sizes.

Perhaps you want to look into this, @he8us?

### PR DESCRIPTION
**Pull request by mielvds**
Fixes #862 with some pragmatic bootstap CSS. Also fixes the dissapearance of the login screen when resizing the popup window.

### CODE REVIEW

File: default-views/auth/login-username-password.hbs [hunk 1]
```diff
@@ -2,18 +2,18 @@
   <div class="panel-body">
     <form method="post" action="/login/password">
       <div class="form-group">
-        <label class="control-label" for="username">Username</label>
-        <input type="text" class="form-control" name="username" id="username" placeholder="alice"/>
+        <label class="control-label hidden-xs" for="username">Username</label>
```
[the remark this answers is not in the document]
Comment (mielvds) [reply default-views/auth/login-username-password.hbs]: Considered that, but I guess the comment would be then that the textfields would be too small on smaller screens. I'll give it a go so I can compare.

[the remark this answers is not in the document]
Comment (megoth) [reply default-views/auth/login-username-password.hbs]: Yes, might not work on small screens.

If `form-horizontal` doesn't work, maybe add another CSS class for this specific form, that lessens padding and margins a bit on smaller screens? I think accessibility with label-input is very important.

File: default-views/auth/login.hbs [hunk 2]
```diff
@@ -8,16 +8,18 @@
   <link rel="stylesheet" href="/common/css/solid.css">
 </head>
 <body>
-<div class="container">
-  <div class="page-header">
+<div class="container-fluid">
+  <div class="page-header hidden-xs hidden-sm">
```
[the remark this answers is not in the document]
Comment (mielvds) [reply default-views/auth/login.hbs]: If the semantic is "Login" then it is not lost, but moved: https://github.com/solid/node-solid-server/blob/cc2d2c596e3182f4a97c513b924cc49b844feb6b/default-views/auth/login.hbs#L22
It's not ideal, but avoids getting into the CSS more.

File: default-views/auth/login-username-password.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (mielvds) [reply default-views/auth/login-username-password.hbs]: Seems ok now

**Comment by megoth** [comment]
@mielvds Could you fix `package-lock.json`, then we'll merge this PR.

**Comment by mielvds** [comment]
should be fine now, don't know why travis fails though

**Pull request merged**